### PR TITLE
Refactor blurr for naming changes

### PR DESCRIPTION
**Pull request by indradeep**
Changed names and terminology to get clarity on Blurr Architecture

### CODE REVIEW

File: blurr/core/loader.py [hunk 1]
```diff
@@ -5,44 +5,46 @@
 from blurr.core.errors import InvalidSchemaError
 
 ITEM_MAP = {
-    'Blurr:DataGroup:BlockAggregate': 'blurr.core.block_data_group.BlockDataGroup',
-    'Blurr:DataGroup:IdentityAggregate': 'blurr.core.identity_data_group.IdentityDataGroup',
-    'Blurr:DataGroup:VariableAggregate': 'blurr.core.variable_data_group.VariableDataGroup',
-    'Blurr:DataGroup:WindowAggregate': 'blurr.core.window_data_group.WindowDataGroup',
+    'Blurr:Transform:Simple': 'blurr.core.transformer_simple.SimpleTransformer',
```
Comment (prashastk) [inline blurr/core/loader.py]: I don't like Blurr:Transform:Simple. Blurr::Transform:Streaming sounds better to me.

File: blurr/core/loader.py [hunk 2]
```diff
@@ -5,44 +5,46 @@
...
+    'datetime': 'blurr.core.field_simple.SimpleField',
+    'float': 'blurr.core.field_simple.SimpleField',
+    'map': 'blurr.core.field_simple.SimpleField',
+    'list': 'blurr.core.field_simple.SimpleField',
+    'set': 'blurr.core.field_simple.SimpleField',
 }
 ITEM_MAP_LOWER_CASE = {k.lower(): v for k, v in ITEM_MAP.items()}
 
 SCHEMA_MAP = {
+    'Blurr:Transform:Simple': 'blurr.core.streaming_transformer.StreamingTransformerSchema',
```
Comment (prashastk) [inline blurr/core/loader.py]: The line below already has the StreamingTransformerSchema

File: docs/Data Transform Config (DTC).md [hunk 3]
```diff
@@ -84,7 +84,7 @@ DataGroups:
 
 Key |  Description | Allowed values | Required
 --- | ------------ | -------------- | --------
-Type | Type of DataGroup | `Blurr:DataGroup:IdentityAggregate`, `Blurr:DataGroup:BlockAggregate`, `Blurr:DataGroup:Variable` | Required
+Type | Type of DataGroup | `Blurr:Aggregate:IdentityAggregate`, `Blurr:Aggregate:BlockAggregate`, `Blurr:Aggregate:Variable` | Required
```
Comment (vigbk) [inline docs/Data Transform Config (DTC).md]: Should be VariableAggregate ?

File: docs/Data Transform Config (DTC).md [hunk 4]
```diff
@@ -123,7 +123,7 @@ A `DataGroup` contains `fields` for the information being stored. IdentityAggreg
...
```
Comment (vigbk) [inline docs/Data Transform Config (DTC).md]: Should be VariableAggregate ?

File: docs/Data Transform Config (DTC).md [hunk 5]
```diff
@@ -141,10 +141,10 @@ When | Boolean expression that defines which raw events to process | Any `boolea
 
 ### Variable
 
-`Blurr:DataGroup:Variable`. Variable DataGroups are temporary variables that can be used in other data blocks. Variables aim to reduce code duplication and improve readability. They are useful for cleansing / modifying / typecasting source elements and representing complex filter conditions that evaluate to a binary value.
+`Blurr:Aggregate:Variable`. Variable DataGroups are temporary variables that can be used in other data blocks. Variables aim to reduce code duplication and improve readability. They are useful for cleansing / modifying / typecasting source elements and representing complex filter conditions that evaluate to a binary value.
```
Comment (vigbk) [inline docs/Data Transform Config (DTC).md]: Should be VariableAggregate ?

File: docs/Data Transform Config (DTC).md [hunk 6]
```diff
@@ -141,10 +141,10 @@ When | Boolean expression that defines which raw events to process | Any `boolea
...
 
 ```yaml
-- Type: Blurr:DataGroup:Variable
+- Type: Blurr:Aggregate:Variable
```
Comment (vigbk) [inline docs/Data Transform Config (DTC).md]: Should be VariableAggregate ?

File: docs/Data Transform Config (DTC).md [hunk 7]
```diff
@@ -260,7 +260,7 @@ DataGroups:
 
 Key |  Description | Allowed values | Required
 --- | ------------ | -------------- | --------
-Type | Type of DataGroup | `Blurr:DataGroup:WindowAggregate`, `Blurr:DataGroup:Variable` | Required
+Type | Type of DataGroup | `Blurr:Aggregate:WindowAggregate`, `Blurr:Aggregate:Variable` | Required
```
Comment (vigbk) [inline docs/Data Transform Config (DTC).md]: Should be VariableAggregate ?

File: blurr/core/loader.py (hunk 1 above)
Comment (indradeep) [reply blurr/core/loader.py]: Accepted.

File: blurr/core/loader.py (hunk 2 above)
Comment (indradeep) [reply blurr/core/loader.py]: Fixed.

**Pull request merged**